logging/logrus/go.mod

```diff
@@ -2,8 +2,11 @@ module github.com/hertz-contrib/obs-opentelemetry/logging/logrus
 
 go 1.17
 
+replace github.com/hertz-contrib/logger/logrus => github.com/CoderPoet/logger/logrus v0.0.0-20221030110852-9f9efbc6c387
```

Comment:
TODO: remove this replace when the [PR](https://github.com/hertz-contrib/logger/pull/4) is merged

Reply:
#4 is done